vars/unitTestPost.groovy

```diff
@@ -44,46 +50,53 @@ def call(Map config = [:]) {
                      allowEmptyArchive: true
     return
   }
+  
+  if (stage_info['with_valgrind'] == 'disabled') {
+    def test_results = config.get('testResults', 'test_results/*.xml')
+    junit testResults: test_results
+  }
```

Comment:
Yeah, so again, why only JUnit results for non-memcheck?  That seems to be adding some unnecessary gating here.